src/server/console/logind_console_services.cpp

```diff
@@ -68,7 +71,7 @@ class GErrorPtr
     }
 
 private:
-    GError* raw_error;
+    GError *raw_error;
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style